### PR TITLE
TOOLS/PERF: Added performance tests for am short iov.

### PR DESCRIPTION
**Pull request by rakhmets**
## What
Added ability to run performance tests for uct_ep_am_short_iov interface.

### CODE REVIEW

File: test/gtest/uct/test_uct_perf.cc [hunk 1]
```diff
@@ -42,6 +42,24 @@ const test_perf::test_spec test_uct_perf::tests[] =
     ucs_offsetof(ucx_perf_result_t, msgrate.total_average), 1e-6, 0.8, 80.0,
```
Comment (dmitrygx) [inline test/gtest/uct/test_uct_perf.cc:42]: minor: add `short` to the name for `UCT_PERF_DATA_LAYOUT_SHORT` tests, e.g. `am short latency`

File: test/gtest/uct/test_uct_perf.cc [hunk 2]
```diff
@@ -42,6 +42,24 @@ const test_perf::test_spec test_uct_perf::tests[] =
...
     0 },
 
+  { "am short iov latency", "usec",
+    UCX_PERF_API_UCT, UCX_PERF_CMD_AM, UCX_PERF_TEST_TYPE_PINGPONG,
+    UCT_PERF_DATA_LAYOUT_SHORT_IOV, 0, 1, { 8, 8 }, 1, 100000lu,
```
Comment (dmitrygx) [inline test/gtest/uct/test_uct_perf.cc:47]: `{ 4, 4 }` to align with short

File: test/gtest/uct/test_uct_perf.cc [hunk 3]
```diff
@@ -42,6 +42,24 @@ const test_perf::test_spec test_uct_perf::tests[] =
...
+    ucs_offsetof(ucx_perf_result_t, latency.total_average), 1e6, 0.01, 2.5,
+    0 },
+
+  { "am short iov rate", "Mpps",
+    UCX_PERF_API_UCT, UCX_PERF_CMD_AM, UCX_PERF_TEST_TYPE_STREAM_UNI,
+    UCT_PERF_DATA_LAYOUT_SHORT_IOV, 0, 1, { 8, 8 }, 1, 2000000lu,
```
Comment (dmitrygx) [inline test/gtest/uct/test_uct_perf.cc:53]: `{ 4, 4 }` to align with short

**Review by dmitrygx — COMMENTED** [review]
👍 besides minor comments

**Review by dmitrygx — APPROVED** [review]

**Review by brminich — APPROVED** [review]

**Comment by brminich** [comment]
bot:pipe:retest

**Review by yosefe — APPROVED** [review]

**Pull request merged**